storage/storage.cpp

```diff
@@ -244,7 +244,7 @@ size_t Storage::GetDownloadedFilesCount() const
 TCountriesContainer const & NodeFromCountryId(TCountriesContainer const & root,
                                               TCountryId const & countryId)
 {
-  SimpleTree<Country> const * node = root.FindLeaf(Country(countryId));
+  SimpleTree<Country> const * node = root.FindFirstLeaf(Country(countryId));
```

Comment:
I cann't do that. SimpleTree is a template class. Here it's used with param Country.